src/quo2/components/avatars/collection_avatar/component_spec.cljs

```diff
@@ -0,0 +1,17 @@
+(ns quo2.components.avatars.collection-avatar.component-spec
+  (:require [quo2.components.avatars.collection-avatar.view :as collection-avatar]
+            [test-helpers.component :as h]))
+
+(defonce bored-ape (js/require "../resources/images/mock2/bored-ape.png"))
```

Comment:
oh if this is a mock image then it should probably be in the preview screens no? 
also I think we generally don't require resources directly into the file, there a few resources file - one in quo2 and one in status-im2 which we use to handle this 👍

Reply:
@J-Son89, do you mean not from the `mock` folder?

Reply:
no that's fine. I mean that we have a quo2.resources file so can we use that instead? and do the `resources/get-mock-image` approach?

Reply:
@J-Son89 ah, okay)

Reply:
@J-Son89, just remembered why it is this way.`resources/get-mock-image` located in `status-im2`, whereas component is in quo2. And our linter fails with warnings when we reference status-im2 from quo2.

Reply:
Oh we have another file like that within quo2 for this reason though

Reply:
@J-Son89, thanks, missed it!)